syntax = "proto3";

package diststore;

service MgmtService {
	rpc GetMembership (GetMembershipRequest) returns (GetMembershipResponse) {}
}
